src/components/BreakoutRoomsEditor/SelectableParticipant.vue

```diff
@@ -15,6 +15,7 @@
 			:source="participant.source || participant.actorType"
 			disable-menu
 			disable-tooltip
+			:preloaded-user-status="preloadedUserStatus"
```

Comment:
It's still a computed prop in `Participant`

Reply:
Yes, but I'll get rid of it in a follow-up